grunt/config/copy.js

```diff
@@ -1,240 +1,207 @@
-module.exports = function (grunt, options) {
+module.exports = function(grunt, options) {
 
-    var _ = require("underscore");
+  var _ = require("underscore");
 
-    var getUnixPath = function(filepath) {
-        // convert to unix style slashes
-        return filepath.replace(/\\/g,"/");
-    };
+  var getUnixPath = function(filepath) {
+    // convert to unix style slashes
+    return filepath.replace(/\\/g, "/");
+  };
 
-    var collate = function(collateAtFolderName, destFolder, srcFileName) {
-        destFolder = getUnixPath(destFolder);
-        srcFileName = getUnixPath(srcFileName);
+  var collate = function(collateAtFolderName, destFolder, srcFileName) {
+    destFolder = getUnixPath(destFolder);
+    srcFileName = getUnixPath(srcFileName);
 
-        // ignore if the srcFileName ends with the collateAtFolderName
-        var nameParts = srcFileName.split("/");
-        if (nameParts[nameParts.length-1] === collateAtFolderName) {
-            return destFolder;
-        }
+    // ignore if the srcFileName ends with the collateAtFolderName
+    var nameParts = srcFileName.split("/");
+    if (nameParts[nameParts.length - 1] === collateAtFolderName) {
+      return destFolder;
+    }
 
-        var startOfCollatePath = srcFileName.indexOf(collateAtFolderName) + collateAtFolderName.length + 1;
-        var collatedFilePath = destFolder + srcFileName.substr(startOfCollatePath);
+    var startOfCollatePath = srcFileName.indexOf(collateAtFolderName) + collateAtFolderName.length + 1;
+    var collatedFilePath = destFolder + srcFileName.substr(startOfCollatePath);
 
-        return collatedFilePath;
-    }
+    return collatedFilePath;
+  }
 
 
-    var nonServerTasks = {
-        courseAssets: {
-            files: [
-                {
-                    expand: true,
-                    src: ['<%= languages %>/**/*', '!**/*.<%= jsonext %>'],
-                    cwd: '<%= sourcedir %>course/',
-                    dest: '<%= outputdir %>course/'
-                }
-            ]
-        },
-        courseJson: {
-            files: [
-                {
-                    expand: true,
-                    src: ['<%= languages %>/*.<%= jsonext %>'],
-                    cwd: '<%= sourcedir %>course/',
-                    dest: '<%= outputdir %>course/'
-                }
-            ]
-        }
-    };
+  var nonServerTasks = {
+    courseAssets: {
+      files: [{
+        expand: true,
+        src: ['<%= languages %>/**/*', '!**/*.<%= jsonext %>'],
+        cwd: '<%= sourcedir %>course/',
+        dest: '<%= outputdir %>course/'
+      }]
+    },
+    courseJson: {
+      files: [{
```

Comment:
@chetan-hajare - Could you put the opening '{' and associated closing '}' on a new line please?